featurebyte/api/target.py

```diff
@@ -0,0 +1,90 @@
+"""
+Target API object
+"""
+from __future__ import annotations
+
+from typing import List, Optional
+
+from pydantic import Field, StrictStr
+
+from featurebyte.api.api_object import ForeignKeyMapping
+from featurebyte.api.entity import Entity
+from featurebyte.api.savable_api_object import SavableApiObject
+from featurebyte.exception import RecordRetrievalException
+from featurebyte.models.base import PydanticObjectId
+from featurebyte.models.target import TargetModel
+from featurebyte.schema.target import TargetUpdate
+
+
+class Target(SavableApiObject):
+    """
+    Target class used to represent a Target in FeatureByte.
+    """
+
+    internal_entity_ids: Optional[List[PydanticObjectId]] = Field(alias="entity_ids")
+    horizon: Optional[StrictStr]
+    blind_spot: Optional[StrictStr]
```

Comment:
We should make `horizon` & `blind_spot` to following `internal_*` pattern too if we allow user to update it.